packages/ai-jsx/.eslintrc.cjs

```diff
@@ -11,6 +11,8 @@ module.exports = {
 
   env: {
     node: true,
+    browser: true,
```

Comment:
We might want to be a little careful with this. This setting will enable browser globals, which means we could write code that depends on browser APIs, which then fails when the code runs in Node.

A more conservative approach would be to enable `env:browser` in just files/directories that actually use it.

But this isn't worth spending more than 60 seconds thinking further about or addressing.